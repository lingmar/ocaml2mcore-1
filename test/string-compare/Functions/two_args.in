let foo x y = x + y in
42