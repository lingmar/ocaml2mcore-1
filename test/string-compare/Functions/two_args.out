mexpr
let foo = lam x. lam y.
  addi x y
in 42
